dll/cpl/desk/lang/ru-RU.rc

```diff
@@ -24,7 +24,7 @@ BEGIN
     LTEXT "Выберите изображение для рабочего стола", IDC_STATIC, 8, 93, 230, 9
     PUSHBUTTON "&Обзор...", IDC_BROWSE_BUTTON, 188, 104, 55, 15
     PUSHBUTTON "&Цвет...", IDC_COLOR_BUTTON, 188, 167, 55, 15
-    LTEXT "Расположение", IDC_STATIC, 188, 128, 55, 9//FIXME: add accel
+    LTEXT "Расположение", IDC_STATIC, 188, 128, 55, 9//FIXME: add accel. I used "&Расположение" for the backport
```

Comment:
```suggestion
    LTEXT "Расп&оложение", IDC_STATIC, 188, 128, 55, 9//FIXME: I used "&Расположение" for the backport
```

Reply:
I cannot use your suggestion, because there is already "&Обзор..." in this dlg.

Reply:
![image](https://github.com/user-attachments/assets/379a0f4f-f345-43b5-8952-15def0e41abb)

Reply:
resolved by https://github.com/reactos/reactos/pull/7298/commits/f426f4166f37a7c7cb3e5e6899f06da11ff098b3 . I will not collide this discussion to keep the picture visible.

Reply:
Meaning, the "&Obzor" accelerator should be fixed in our dialog.
Or alternatively, we don't blindly stick with Windows XP accelerators here, and put the accelerators in the first letters here (since this is possible: "&Obzor", "&Raspolojenye", "&Tsvet" .

Reply:
Your translation doesn't make any sense in Portuguese. The two-line solution solves the issue well.